examples/server-alarms-and-conditions.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+
+from aioconsole import ainput
+import logging
+
+import asyncio
+
+from asyncua import Server, ua
+logging.basicConfig(level=logging.WARNING)
+
+class OpcUaServer(object):
+
+    def __init__(self, endpoint):
+        self.server = Server()
+        self.server.set_endpoint(endpoint)
+        self.server.set_server_name('Alarms and Conditions Test Server')
+        self.server.application_type = ua.ApplicationType.Server
+        self.server.iserver.bind_condition_methods = 500
+        self.con_gen = None
+        self.alarm_gen = None
+
+    async def __aenter__(self):
+        await self.init()
+        await self.server.start()
+        return self
+
+    async def __aexit__(self, exc_type, exc_value, traceback):
+        await self.server.stop()
+
+    async def init(self):
+        await self.server.init()
+        uri = "http://examples.freeopcua.github.io"
+        idx = await self.server.register_namespace(uri)
+        objects = self.server.get_objects_node()
+
+        noti_node = await objects.add_object(idx, 'NotifierObject')
+
+        con_obj = await noti_node.add_object(idx, "ConditionObject")
+        condition = self.server.get_node(ua.NodeId(2830))
+        self.con_gen = await self.server.get_event_generator(condition, self.server.nodes.server,
+                                                             notifier_path=[ua.ObjectIds.Server])
+        self.con_gen.event.add_property('NodeId', con_obj.nodeid, ua.VariantType.NodeId)
+
+        alarm_obj = await noti_node.add_object(idx, "AlarmObject")
+        alarm = self.server.get_node(ua.NodeId(10637))
+        self.alarm_gen = await self.server.get_event_generator(alarm, self.server.nodes.server,
+                                                               notifier_path=[ua.ObjectIds.Server])
+
+    async def generate_condition(self, retain):
+        self.con_gen.event.ConditionName = 'Example Condition'
+        self.con_gen.event.Message = ua.LocalizedText("Some Message")
```

Comment:
Add missing ConditionId:
```suggestion
    async def generate_condition(self, retain):
        self.con_gen.event.NodeId = ua.NodeId("Id111_EC") # Unique ConditionId
        self.con_gen.event.ConditionName = 'Example Condition'
        self.con_gen.event.Message = ua.LocalizedText("Some Message")
```